CompetitionTemplate/src/competition/subsystems/vision/commands/SpewVisionInformationCommand.java

```diff
@@ -0,0 +1,33 @@
+package competition.subsystems.vision.commands;
+
+import org.apache.log4j.Logger;
+
+import com.google.inject.Inject;
+
+import competition.subsystems.vision.VisionSubsystem;
+import xbot.common.command.BaseCommand;
+
+public class SpewVisionInformationCommand extends BaseCommand {
+    static Logger log = Logger.getLogger(SpewVisionInformationCommand.class);
+    
+    private VisionSubsystem visionSubsystem;
+    
+    @Inject
+    public SpewVisionInformationCommand(VisionSubsystem visionSubsystem) {
+        this.visionSubsystem = visionSubsystem;
+        requires(visionSubsystem);
+        setRunWhenDisabled(true);
+    }
+    
+    @Override
+    public void initialize() {
+
+    }
+
+    @Override
+    public void execute() {
```

Comment:
To prevent flooding of our own logs, I suggest throttling this to only do work on every 50 executions or something.